hermes.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Main program loop for Hermes. This initialises Cloudprober so that Hermes can
+// interact with it through gRPCs.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/google/cloudprober"
+	"github.com/google/cloudprober/web"
+	"github.com/googleinterns/step224-2020/hermes"
+)
+
+var (
+	rpc_port = flag.Int("rpc_port", 9314, "The port that the gRPC server of Cloudprober will run on.")
```

Comment:
nit:  Flag names in Go binaries should prefer to use underscores to separate words, though the variables that hold a flag's value should follow the standard Go name style (mixed caps). Specifically, the flag name should be in snake case, and the variable name should be the equivalent name in camel case.
```suggestion
	rpcPort = flag.Int("rpc_port", 9314, "The port that the gRPC server of Cloudprober will run on.")
```

Reply:
That's interesting, thank you. I've updated this file with this change.